tests/test_properties.py

```diff
@@ -590,7 +590,7 @@ def test_date_docprops_with_format_get_updated(self):
 
         CustomProperties(document).update_all()
 
-        expected_values = [u'23.01.20', u'Thursday 23 January 2020', u'23-1-20 10:0:0']
+        expected_values = [u'23.01.20', u'jeudi 23 janvier 2020', u'23-1-20 10:0:0']
```

Comment:
Just wonder if this (implicit) test is enough or if we should add an explicit test with a dedicated `.docx` only for the language specific dates...